pkg/lib/infra/sms/custom/model.go

```diff
@@ -12,3 +14,19 @@ type SendOptions struct {
 	LanguageTag       string                    `json:"language_tag"`
 	TemplateVariables *smsapi.TemplateVariables `json:"template_variables"`
 }
+
+// See https://github.com/authgear/authgear-sms-gateway/blob/main/pkg/handler/api.go
+type ResponseBody struct {
+	Code           string  `json:"code"`
+	ErrorDetail    *string `json:"error_detail,omitempty"`
```

Comment:
It is named `error_description` there.

Reply:
I am planning to add a new field in the response body, so that the sms gateway can return detail about the error which I we can display it in the ui. For example, the error code returned from the gateway.

Reply:
As discussed it will be replaced by

```
	ProviderName      string `json:"provider_name,omitempty"`
	ProviderErrorCode string `json:"provider_error_code,omitempty"`
```